Cakefile

```diff
@@ -228,11 +228,15 @@ coffee = do ->
   # Will *always* produce both concatenated and minified
   # versions if concat is true.
   compile = (files, output, concat, callback) ->
-    cmd = "#{config.coffee} --compile"
-    cmd += " --join #{concat}.js" if concat
-    files = q(files).join " "
-    cmd += " --output #{q output} #{files}"
-    exec cmd, (err, stdout, stderr) ->
+    # "cat" (read the contents of) all files
+    catCmd = "cat #{files}"
```

Comment:
```cat``` is available in Windows' Powershell by default.
But it seems like the ```exec fullCmd``` executes in the normal batch environment, where ```cat``` is not normally available, even when run from Powershell.

So you do need another solution to make this cross platform compatible.